client/navigation/components/container/index.js

```diff
@@ -86,7 +86,11 @@ const Container = ( { menuItems } ) => {
 						title={ category.title }
 						menu={ category.id }
 						parentMenu={ category.parent }
-						backButtonLabel={ category.backButtonLabel }
+						backButtonLabel={
```

Comment:
Is this really needed?

If it is, `backButtonLabel={ category.backButtonLabel || null }` would be more succinct and easier to grok.

Reply:
It is needed to show "WooCommerce Home" that is defined by the server, but agreed that your recommended syntax is more succinct- thanks for the suggestion!